components/brave_wallet_ui/components/desktop/popup-modals/edit-visible-assets-modal/index.tsx

```diff
@@ -94,7 +95,11 @@ const EditVisibleAssetsModal = (props: Props) => {
   }, [fullAssetList, userVisibleTokensInfo])
 
   React.useMemo(() => {
```

Comment:
Is `React.useEffect` more appropriate here?

Reply:
Updated!